src/Bicep.Core.Samples/Files/Modules_CRLF/main.json

```diff
@@ -1,1829 +1,1914 @@
-{
```

Comment:
this is odd...

Reply:
This should to be due to line endings normalization which is expected.